File: app/lib/model/action.dart
class Ations {
  String id;
  String name;
  String img;
  String categoryId;


  Ations({this.id, this.name, this.img, this.categoryId});

  Ations.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        name = json['name'],
        img = json['img'],
        categoryId = json['categoryId'];

  Map<String, dynamic> toJson() => {
    'id': id,
    'name': name,
    'img': img,
    'categoryId' : categoryId,
  };
}

File: be/internal/entity/action.go
package entity

import "github.com/google/uuid"

type Action struct {
	Base

	Name        string  `gorm:"type:varchar(128); not null" validate:"required,max=128" json:"name"`
	Img 		string `gorm:"type:varchar(10485760); not null" json:"img"`
	CategoryId 	uuid.UUID  `gorm:"type:uuid; not null" json:"categoryId"`
	UserId 		uuid.UUID  `gorm:"type:uuid; not null" json:"userId"`
}